### PR TITLE
Introducing Harvester Guru on Gurubase.io

### PR DESCRIPTION
**Pull request by kursataktas**
Hello team,

I'm the maintainer of [Anteon](https://github.com/getanteon/anteon). We have created Gurubase.io with the mission of building a centralized, open-source tool-focused knowledge base. Essentially, each "guru" is equipped with custom knowledge to answer user questions based on collected data related to that tool.

I wanted to update you that I've manually added the [Harvester Guru](https://gurubase.io/g/harvester) to Gurubase. Harvester Guru uses the data from this repo and data from the [docs](https://docs.harvesterhci.io/) to answer questions by leveraging the LLM.

In this PR, I showcased the "Harvester Guru", which highlights that Harvester now has an AI assistant available to help users with their questions. Please let me know your thoughts on this contribution.

Additionally, if you want me to disable Harvester Guru in Gurubase, just let me know that's totally fine.

### CODE REVIEW

**Review by ibrokethecloud — APPROVED** [review]

**Comment by asettle** [comment]
Thanks! This is pretty cool.

**Comment by innobead** [comment]
@kursataktas this looks interesting and helpful for the community. 

I have a few questions.
- How can we determine which version of the data Guru is using? Is it pulled from all versions of the document?  
- Is there a dev portal available for configuring the sources collected by Guru? This would help us ensure that the correct data sources are being used together with LLM, allowing for more accurate responses for users of your service.

**Comment by kursataktas** [comment]
Thanks for the review, @innobead!

I manually added the Harvester v1.3 documentation webpages as a data source. Currently, Gurubase does not support versioning. This would be a great feature to have in the future. At present, it supports only one version by adding that version's documentation pages as the data source.

Yes, you can manage Harvester Guru on Gurubase. You can find the details [here](https://github.com/getanteon/gurubase?tab=readme-ov-file#how-to-claim-a-guru).

**Comment by ihcsim** [comment]
@kursataktas This is great! Can we move the href to the 'community' section of the README for now, until we have a chance to look into managing the hv guru? So something like:

```diff
diff --git a/README.md b/README.md
index c5cf10d8f..1cadeef4b 100644
--- a/README.md
+++ b/README.md
@@ -128,6 +128,7 @@ If you need any help with Harvester, please join us at either our [Slack](https:
 
 If you have any feedback or questions, feel free to [file an issue](https://github.com/harvester/harvester/issues/new/choose).
 
+Harvester is on [gurubase.io](https://gurubase.io/g/harvester)!
 
 ## License
 Copyright (c) 2024 [Rancher Labs, Inc.](http://rancher.com)
```

**Comment by kursataktas** [comment]
@ihcsim I’ve updated the PR based on your review and made some changes to the text. What do you think?

**Comment by ihcsim** [comment]
@kursataktas What's the scope of the the license grant (clause 3.1) defined in gurubase ToU? Does that also include materials derived from the Harvester documentation?

**Comment by kursataktas** [comment]
> @kursataktas What's the scope of the the license grant (clause 3.1) defined in gurubase ToU? Does that also include materials derived from the Harvester documentation?

The license grant in clause 3.1 applies specifically to content that users submit directly to Gurubase. For example, this could include a comment you leave on an article or a post you create on Gurubase (even though we don’t currently have a post-creation feature).

Any materials derived from third-party sources, such as the Harvester documentation, remain subject to the original licensing terms of those sources. Gurubase adheres to these terms and ensures proper compliance and attribution, as outlined in section 2.2 of the Terms of Use.

**Review by bk201 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ If you need any help with Harvester, please join us at either our [Slack](https:
 
 If you have any feedback or questions, feel free to [file an issue](https://github.com/harvester/harvester/issues/new/choose).
 
+You can also [Ask Harvester Guru](https://gurubase.io/g/harvester), it is a Harvester-focused AI to answer your questions.
```
Comment (ihcsim) [inline README.md]:
```suggestion
You can also [ask Harvester Guru](https://gurubase.io/g/harvester) your questions.
```
I think "Harvester-focused AI" might mean different things to different people.

**Review by ihcsim — APPROVED** [review]

**Comment by kursataktas** [comment]
Hello Everyone,

We recently released the embeddable "Ask AI" widget, which allows you to integrate your Guru’s Q&A capability into websites, documentation, and more.

The installation process is straightforward, as explained in [this GitHub repository](https://github.com/getanteon/gurubase-widget).

You can see it in action on the [Anteon](https://getanteon.com/) and [NetAlertX](https://netalertx.com/) websites.

**Comment by kursataktas** [comment]
Hello Everyone,

I want to share an exciting update about Gurubase. We've recently released several great features that I think you'll find interesting.

### 1. Slack & Discord Integration
You can now add your Guru as a bot to Slack and Discord, allowing you to ask questions instantly. More details here: [Discord Bot Integration](https://docs.gurubase.ai/integrations/discord-bot).

### 2. Analytics Dashboard
Track what your users ask your Guru and gain valuable insights. This feature is available on the "My Gurus" page. If you haven’t maintained your Guru yet, I highly recommend doing so to access your analytics. More information about claiming your Guru can be found [here](https://github.com/Gurubase/gurubase?tab=readme-ov-file#how-to-claim-a-guru).
image

### 3. API Support
All Guru functionalities are now accessible via API. You can ask questions, edit data sources, and more using our API endpoints. Here is the [API Doc](https://docs.gurubase.ai/api-reference/introduction).

### 4. Codebase Indexing
We now support indexing codebases from GitHub repositories to generate answers for related queries. This feature is available for select Gurus. If you don’t see the "Codebase Indexed" label on your Guru edit page and would like access, please let me know, I’d be happy to enable it for you!
image

Let me know if you have any questions or feedback!

**Pull request merged**